src/bo4e/bo/marktteilnehmer.py

```diff
@@ -1,15 +1,18 @@
 import attr
-import jsons
 from attr.validators import matches_re
 
-from bo4e.bo.geschaeftspartner import Geschaeftspartner
+from marshmallow import fields
+from marshmallow_enum import EnumField
+
+from bo4e.bo.geschaeftspartner import Geschaeftspartner, GeschaeftspartnerSchema
+from bo4e.cases import JavaScriptMixin
 from bo4e.enum.botyp import BoTyp
 from bo4e.enum.marktrolle import Marktrolle
 from bo4e.enum.rollencodetyp import Rollencodetyp
 
 
 @attr.s(auto_attribs=True, kw_only=True)
-class Marktteilnehmer(Geschaeftspartner, jsons.JsonSerializable):
+class Marktteilnehmer(Geschaeftspartner):
```

Comment:
Den Docstring und die Kommentare welche attributes required sind oder nicht, hast du nur hier in den BOs. Ist das Absicht oder fehlen die noch bei den anderen BOs?